spacy/tokens/span_group.pyx

```diff
@@ -103,6 +106,10 @@ cdef class SpanGroup:
 
         DOCS: https://spacy.io/api/spangroup#getitem
         """
+        if isinstance(i, slice):
+            start, stop = util.normalize_slice(len(self), i.start, i.stop, i.step)
+            spans = [self[i] for i in range(start, stop)]
+            return SpanGroup(self.doc, spans=spans)
```

Comment:
This is probably not the fastest way to do this.

Reply:
And I suppose we could support `step` here, but I'm not sure why you'd want to do this.